jni/src/main/java/com/horizen/vrfnative/VRFPublicKey.java

```diff
@@ -32,6 +32,10 @@ public static VRFPublicKey deserialize(byte[] publicKeyBytes, boolean checkPubli
     return nativeDeserializePublicKey(publicKeyBytes, checkPublicKey);
   }
 
+  public static VRFPublicKey deserialize(byte[] publicKeyBytes) {
```

Comment:
You need this default "deserialize" only for SchnorrPk, SchnorrSignature and VrfPk ? Nothing else ? Like SNARK pk, vk, proof, or Vrf proof and so on

Reply:
I forgot about vrf proof. Will add it.
About snark proofs and keys it is fine to keep as is.